src/Platform.Handlers/src/Xamarin.Platform.Handlers/Handlers/Label/LabelHandler.Android.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Android.Widget;
+using Xamarin.Forms;
+
+namespace Xamarin.Platform.Handlers
+{
+	public partial class LabelHandler : AbstractViewHandler<ILabel, TextView>
+	{
+		static Color DefaultTextColor { get; set; }
```

Comment:
Just adding a note for the future me. This is fine for a few reasons:
 - this should always be the same value
 - it only ever runs once because the base has a check and runs it once per handler